src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,61 @@
-export const PeopleFilters = () => {
+/* eslint-disable no-restricted-syntax */
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { Filter } from '../types/Filter';
+import { getSearchWith } from '../utils/searchHelper';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get(Filter.SEX) || null;
+  const name = searchParams.get(Filter.NAME) || '';
+  const centuries = searchParams.getAll('centuries') || null;
+
+  const centuriesValues = [16, 17, 18, 19, 20];
+
+  const resetParams = () => {
+    const newSearchParams = new URLSearchParams('');
+
+    return newSearchParams.toString();
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function setParams(params: any) {
```

Comment:
using `any` is not a good practice. Specify what exactly you will get (or write a generic)